examples/axisartist/demo_axisline_style.py

```diff
@@ -3,7 +3,10 @@
 Axis line styles
 ================
 
-This example shows some configurations for axis style.
+This example shows some configurations for axis style. 
+
+Note: If you want to obtain arrow heads at the ends of the axes, also check
+out the :doc:`/gallery/recipes/centered_spines_with_arrows` example.
```

Comment:
I would suggest warning more explicitly against using axisartist unless you know you really need it, given its limitations.